app/controllers/register.js

```diff
@@ -60,7 +57,9 @@ export default class RegisterController extends Controller {
             await this.store.findRecord('user', tokenPayload.identity)
           );
         }
-        this.transitionToRoute('public.role-invites', eventId, { queryParams: { token } });
+        if (token != null) {
+          this.transitionToRoute('public.role-invites', eventId, { queryParams: { token } });
+        }
```

Comment:
Shouldn't the `this.transitionToRoute('/');` call be placed here instead in else condition

Reply:
ok